src/api/Feed.js

```diff
@@ -184,14 +185,73 @@ export const getParsedFileActivitiesResponse = (response?: { entries: FileActivi
                     return { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
                 }
 
+                case FILE_ACTIVITY_TYPE_VERSION: {
+                    const versionsItem = { ...source[FILE_ACTIVITY_TYPE_VERSION] };
+
+                    versionsItem.type = 'file_version';
+                    if (versionsItem.action_by) {
+                        const collaborators = {};
+
+                        if (versionsItem.action_by.length === 1) {
+                            versionsItem.uploader_display_name = versionsItem.action_by[0].name;
+                        }
+
+                        versionsItem.action_by.map(collaborator => {
+                            collaborators[collaborator.id] = { ...collaborator };
+                            return collaborator;
+                        });
+
+                        versionsItem.collaborators = collaborators;
+                    }
+                    if (versionsItem.end?.number !== undefined) {
```

Comment:
is this okay?
```suggestion
                    if (versionsItem.end?.number) {
```
same for `versionsItem.start`

Reply:
Yeah it should be okay, version number shouldn't be 0